src/games/kalambury/Board/GameBoard.js

```diff
@@ -27,6 +28,29 @@ const GameBoard = ({ guess, setGuess, envokeLastAnswer, guessInputRef }) => {
     success: false,
   };
 
+  const throttledSendAppend = useMemo(
+    () =>
+      throttleAccumulate(sendChatMessage, 200, ([acc], [args]) => [
+        {
+          type: acc.type,
+          data: [...acc.data, ...args.data],
+        },
+      ]),
+    [sendChatMessage]
+  );
+
+  // send full drawing every 1sec
+  useEffect(() => {
+    if (isDrawing) {
+      const interval = setInterval(() => {
+        throttledSendAppend.flush();
+        sendChatMessage({ type: "UpdateDrawing:replace", data: lines });
+      }, 1000);
```

Comment:
This interval is separate form the one where we only send partial updates. Doesn't it mean that every second there will be 2 (roughly) simultaneous requests one with partial update and one with full drawing. Maybe we could somehow skip this extra partial update (so every 5th one in this case)? 👀

Reply:
Potentially we could, but it would be really hard to sync them. And that would only happen when someone is constantly drawing. I think that sending 2 updates in the same time over websocket is not that big deal.

Reply:
Ok, let's leave it like this for now